executor/aggfuncs/func_percent_rank.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aggfuncs
+
+import (
+	"github.com/pingcap/tidb/sessionctx"
+	"github.com/pingcap/tidb/util/chunk"
+)
+
+// percentRank is the window function to calculate the percentile ranking of rows in a result set.
```

Comment:
how about:
```suggestion
// percentRank calculates the percentage of partition values less than the value in the current row, excluding the highest value.
```

Reply:
In window functions, we usually use 'partition' instead of 'result set'. Refer to the mysql document: https://dev.mysql.com/doc/refman/8.0/en/window-function-descriptions.html#function_percent-rank